packages/sogrim-app/src/components/Pages/SemestersPage/SemesterGrid/course-validator.ts

```diff
@@ -29,7 +25,13 @@ const validCourseCredit = (credit: string | number) => {
 
 const validGrade = (grade: any) => {
   const gradeNumber = Number(grade);
-  if (grade === "" || grade === 0 || grade === "0" || grade === "-") {
+  if (
+    grade === undefined ||
```

Comment:
did you validate this? last time I added the check for "undefined" it caused some weird bugs when I tried adding\editing courses

Reply:
Yes I did. And it seems fine.
Do you have a concrete example so I can test it?

Reply:
nothing specific, if you validated this it's ok from my end 👍